### DISCUSSION TITLE
Focus window on scrolling interaction on whole window, rather than just window border

### DISCUSSION THREAD

**Discussion by MxEggSalad** [i3/i3#6271] [Q&A] [answered]
Basically i just want to focus on my different windows by scrolling on them. 
This is already *kind of* a feature, via scrolling on the border, but being only a few pixels large its kind of hard to hit them. 

Just wondering if theres any hacky solution anyone else has or a proper feature i can put in my config

**Comment by okraits** [comment]
Did you try to make it work configuring the mouse bindings? https://i3wm.org/docs/userguide.html#mousebindings

The general problem:

If i3 consumes the mouse scroll events, then your programs will not receive them anymore. Which means that scrolling won't work anymore in your browser, your file manager, your pdf reader, etc.

**Reply by MxEggSalad** [reply]
oughh would hurt to use that as a solution
might just do `mod$ + scroll` or whatever the keycode is, as a standin

**Reply by MxEggSalad** [reply, marked as answer]
okay weird but i added the lines 
```
# Focus with mouse scroll
bindsym --whole-window $mod+button4 focus
bindsym --whole-window $mod+button5 focus
```
to my config and now just scrolling works to change focus. really dont know why that works ¯\_(ツ)_/¯
